torch/csrc/Module.cpp

```diff
@@ -2063,6 +2069,26 @@ Call this whenever a new thread is created in order to propagate values from
             at::DataPtr(reinterpret_cast<void*>(data_ptr), device));
       });
 
+  py_module.def(
+      "_get_fp32_precision",
+      [](std::string backend = "generic", std::string op = "all") {
```

Comment:
Do we need default values here? Seems like you're always passing them anyways?

Reply:
Removed all unnecessary default values, thanks for advice.